region/norfair/east.json

```diff
@@ -385,6 +385,21 @@
                     "It is possible to shoot upwards from the ground then immediately press up and jump to shinespark without delay.",
                     "A Charge shot can help, as it slows down the shot."
                   ]
+                },
+                {
+                  "name": "Cathedral Entrance Speedjump (Right to Left)",
+                  "notable": true,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canTrickyDashJump",
+                    "canWalljump",
```

Comment:
Can't walljump since you have to shoot the block out.  
Downgrab makes this a little easier.